### PR TITLE
[NOREF] - Condensed task list queries, deprecated unneeded useQuery hook

### PR DESCRIPTION
**Pull request by patrickseguraoddball**
# NOREF

## Changes and Description

- Condensed `GetModelPlanCollabotors` and `GetModelPlan` queries on task list
- Deprecated unneeded `useQuery` hook in favor of `useContext` within `ModelInfoWrapper`

## How to test this change

- Run FE tests
- Check data is intact with collaborators in task list

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: src/views/ModelPlan/Documents/index.tsx [hunk 1]
```diff
@@ -40,16 +34,7 @@ export const DocumentsContent = () => {
...
-    GetModelPlan,
-    {
-      variables: {
-        id: modelID
-      }
-    }
-  );
-
-  const modelPlan = data?.modelPlan || ({} as GetModelPlanTypes);
+  const { modelName } = useContext(ModelInfoContext);
```
Comment (garyjzhao) [inline src/views/ModelPlan/Documents/index.tsx:38]: 🙌🏻 🎉

File: src/views/ModelPlan/TaskList/index.tsx [hunk 2]
```diff
@@ -29,11 +29,11 @@ import PageHeading from 'components/PageHeading';
...
-import GetModelPlanCollaborators from 'queries/Collaborators/GetModelCollaborators';
-import {
-  GetModelCollaborators,
-  GetModelCollaborators_modelPlan_collaborators as GetCollaboratorsType
-} from 'queries/Collaborators/types/GetModelCollaborators';
+// import GetModelPlanCollaborators from 'queries/Collaborators/GetModelCollaborators';
+// import {
+//   GetModelCollaborators,
+//   GetModelCollaborators_modelPlan_collaborators as GetCollaboratorsType
+// } from 'queries/Collaborators/types/GetModelCollaborators';
```
Comment (garyjzhao) [inline src/views/ModelPlan/TaskList/index.tsx:36]: Found some commented code

File: src/views/ModelPlan/Documents/__snapshots__/index.test.tsx.snap [hunk 3]
```diff
@@ -110,24 +110,306 @@ exports[`Model Plan Documents page matches snapshot 1`] = `
           Add a document
         </a>
         <div
-          class="margin-y-10"
-          data-testid="page-loading"
+          class=""
```
Comment (garyjzhao) [inline src/views/ModelPlan/Documents/__snapshots__/index.test.tsx.snap:113]: nit: I recognize this is a snapshot, but found it odd that there is a empty classname

**Review by garyjzhao — CHANGES_REQUESTED** [review]
Nice work. 
Found some things to clean up in the code

File: src/views/ModelPlan/TaskList/index.tsx (hunk 2 above)
Comment (patrickseguraoddball) [reply src/views/ModelPlan/TaskList/index.tsx:36]: Ooops, thanks!

**Review by garyjzhao — APPROVED** [review]
Looks good

**Pull request merged**